[docs] Add "Persian" to the list of RTL languages (#20679)
mui-org/material-ui: docs/src/pages/guides/right-to-left/right-to-left.md

# Right-to-left

> <p class="description">To change the direction of Material-UI components you must follow the following steps. UIs for languages that are read from right-to-left (RTL), such as Arabic and Persian and Hebrew, should be mirrored.</p>

Reviewer: ```suggestion
<p class="description">Right-to-left languages such as Arabic, Persian or Hebrew are supported. To change the direction of Material-UI components you must follow the following steps.</p>
```
